docs/src/modules/ROOT/pages/optimization-algorithms/construction-heuristics.adoc

```diff
@@ -702,6 +702,76 @@ is supported too.
 For scaling out, see <<scalingConstructionHeuristics,scaling construction heuristics>>.
 
 
+[#mixedModelConstructionHeuristics]
+== Mixed modeling and construction heuristics
```

Comment:
I wonder if this should be here now.
Yes, it should be here when the feature is fully delivered, but now when it only works for CH? Maybe let's not tell people to start using it, if they can really only use it for CH and not for LS.